CONTRIBUTING.md

```diff
@@ -52,10 +52,55 @@ Pull requests often need some real-world testing.
 
 2. Run `npm install`.
 
-3. Go to the `webpack-dev-server` module (`cd node_modules/webpack-dev-server`), and run `npm run prepublish`.
+3. Go to the `webpack-dev-server` module (`cd node_modules/webpack-dev-server`), and run `npm run prepare`.
 
 The pull request is now ready to be tested.
 
+
+## Commit message
+
+Our commit messages format follows the [angular.js commits format](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format).
+
+We don't use the scope. The template of a commit would look like this:
+
+### Commit Message Format
+
+Each commit message consists of a **header**, a **body** and a **footer**. The header has a special
+format that includes a **type** and a **subject**:
+
+```md
+<type>: <subject>
+<BLANK LINE>
+
+<body>
+<BLANK LINE>
+<footer>
+```
+
+This is the list of _type_ of commits that we accept:
+
+-   ast
+-   break
+-   chore
+-   cli
+-   docs
+-   feat
+-   fix
+-   misc
+-   tests
```

Comment:
We use conventional commits so our type is other https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional

Reply:
done 👍